pisa/utils/config_parser.py

```diff
@@ -528,13 +528,16 @@ def parse_param(config, section, selector, fullname, pname, value):
     return param
 
 
-def parse_pipeline_config(config):
+def parse_pipeline_config(config,skip_stages=None):
     """Parse pipeline config.
 
     Parameters
     ----------
     config : string or ConfigParser
 
+    skip_stages: None or List of Tuples identifying
```

Comment:
one space before colon; following is more precise type specification:
`sequence of 2 strings, sequence of these, or None; optional`